sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/ingestion/models/CustomPropertiesLog.java

```diff
@@ -113,7 +115,7 @@ public int hashCode() {
     private static Map<String, Object> readProperties(JSONObject object) throws JSONException {
         JSONArray jArray = object.optJSONArray(PROPERTIES);
         if (jArray == null)
-            return null;
+            throw new JSONException("Properties not found");
```

Comment:
Replace those 3 lines by `JSONArray jArray = object.getJSONArray(PROPERTIES)` to get the same behavior.